src/Validation/Validator.php

```diff
@@ -771,7 +785,8 @@ public function allowEmpty($field, $when = true, $message = null)
      * ```
      *
      * @param string $field The name of the field.
-     * @param int|null $flags A bitmask of EMPTY_* flags which specify what is empty
+     * @param int|null $flags A bitmask of EMPTY_* flags which specify what is empty.
```

Comment:
This could technically default to EMPTY_NULL now.

Reply:
Yes, but if for e.g. someone wants to pass the 3rd or 4th argument and wants to keeps flags to default that using `null` is more convenient than `Validator::EMPTY_NULL`.